### PR TITLE
Bump psr/log and psr/http-client

### PR DESCRIPTION
**Pull request by rullzer**
Signed-off-by: Roeland Jago Douma <roeland@famdouma.nl>

### CODE REVIEW

File: psr/log/Psr/Log/LoggerInterface.php [hunk 1]
```diff
@@ -113,11 +113,13 @@ public function debug($message, array $context = array());
...
-     * @param mixed  $level
-     * @param string $message
-     * @param array  $context
+     * @param mixed   $level
+     * @param string  $message
+     * @param mixed[] $context
      *
      * @return void
+     *
+     * @throws \Psr\Log\InvalidArgumentException
```
Comment (ChristophWurst) [inline psr/log/Psr/Log/LoggerInterface.php:122]: :eyes:

File: composer/autoload_psr4.php [hunk 2]
```diff
@@ -75,10 +72,8 @@
     'Ds\\' => array($vendorDir . '/php-ds/php-ds/src'),
     'Doctrine\\DBAL\\' => array($vendorDir . '/doctrine/dbal/lib/Doctrine/DBAL'),
     'Doctrine\\Common\\Lexer\\' => array($vendorDir . '/doctrine/lexer/lib/Doctrine/Common/Lexer'),
-    'Doctrine\\Common\\Inflector\\' => array($vendorDir . '/doctrine/inflector/lib/Doctrine/Common/Inflector'),
     'Doctrine\\Common\\Cache\\' => array($vendorDir . '/doctrine/cache/lib/Doctrine/Common/Cache'),
-    'Doctrine\\Common\\Annotations\\' => array($vendorDir . '/doctrine/annotations/lib/Doctrine/Common/Annotations'),
-    'Doctrine\\Common\\' => array($vendorDir . '/doctrine/common/lib/Doctrine/Common', $vendorDir . '/doctrine/event-manager/lib/Doctrine/Common', $vendorDir . '/doctrine/persistence/lib/Doctrine/Common', $vendorDir . '/doctrine/reflection/lib/Doctrine/Common'),
+    'Doctrine\\Common\\' => array($vendorDir . '/doctrine/event-manager/lib/Doctrine/Common'),
```
Comment (ChristophWurst) [inline composer/autoload_psr4.php:76]: the `React`, `GuzzleHttp` and `Doctrine` changes are unexpected

Comment (ChristophWurst) [reply composer/autoload_psr4.php:76]: it's also weird when I reset the file and do the vendoring locally …

**Review by ChristophWurst — APPROVED** [review]

**Review by MorrisJobke — APPROVED** [review]

**Comment by MorrisJobke** [comment]
CI says "no"

**Comment by ChristophWurst** [comment]
ai, I think because of composer v2. it does some platform checks

**Review by ChristophWurst — APPROVED** [review]
Rebased

**Comment by ChristophWurst** [comment]
CI failure is expected, it's the usual hashes that don't match

**Pull request merged**